### ISSUE TITLE
Release script fails with "Cannot find module '@aws-sdk/middleware-sdk-sqs'"

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#798]
**Describe the bug**
Release script fails with "Cannot find module '@aws-sdk/middleware-sdk-sqs'"

**To Reproduce (observed behavior)**
* Run internal release scripts on `smithy-codegen`
* Release script fails with:
```console
lerna ERR! yarn run pretest exited 2 in '@aws-sdk/client-sqs'
--
lerna ERR! yarn run pretest stdout:
$ tsc
commands/ReceiveMessageCommand.ts(11,41): error TS2307: Cannot find module '@aws-sdk/middleware-sdk-sqs'.
commands/SendMessageBatchCommand.ts(14,43): error TS2307: Cannot find module '@aws-sdk/middleware-sdk-sqs'.
commands/SendMessageCommand.ts(11,38): error TS2307: Cannot find module '@aws-sdk/middleware-sdk-sqs'.
```

**Expected behavior**
New versions of packages are released

**Comment by srchase** [comment]
Fixed in #802

**Comment by trivikr** [comment]
The PR which added middleware https://github.com/aws/aws-sdk-js-v3/pull/719
It was not added as a dependency when CodeGen was run, as we our generate-clients script doesn't override `package.json` - if it's already present https://github.com/aws/aws-sdk-js-v3/blob/80e6356d329151138e144cc75b3abb8a142a83c3/scripts/generate-clients/copy-to-clients.js#L7-L17

We're manually adding dependencies as of now, as done in #802
A long-term fix is discussed in #804
